source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
+with the MongoDB Java Driver. 
+
+Geospatial data is data tied to a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with the **GeoJSON**
+format. GeoJSON is a geospatial data format. You can use the GeoJSON format to
+store your data, and you must use the GeoJSON format to query your geospatial
+data with MongoDB. 
+
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
+      ]
+   }
+
+For definitive information on GeoJSON, see the
+`official IETF GeoJSON specification <https://tools.ietf.org/html/rfc7946>`__.
```

Comment:
suggestion: drop "IETF GeoJSON". IETF isn't previously defined, and the subject (GeoJSON) is already established.

```suggestion
`official specification <https://tools.ietf.org/html/rfc7946>`__.
```

Reply:
**suggestion:** You could also use :rfc:`official specification <7946>`.

Reply:
Thanks for the feedback Nathan and Tony.I didn't realize we had an IETF RFC directive, good to know. 

Chris C had an interesting piece of feedback in our team PR review stating that "the reader might interpret this specification as having something to do with MongoDB." I thought this was valid, as when researching this page it took me a little bit to understand that GeoJSON was not a MongoDB spec. I agree with Chris' feedback, and think it could benefit some readers to clarify in this link who made the specification (IETF).

I also agree with your feedback Nathan that the word "GeoJSON" is extraneous, but I support leaving IETF in. I have added a definition for IETF earlier in the section. Please let me know if you disagree with this assessment, still prefer I remove "IETF" after hearing this defense of keeping it in. 

Fix: https://github.com/mongodb/docs-java/pull/78/commits/7ea4ae1718fcaec2992efa934e276249ea390582